aggregator/src/app/EthereumService.ts

```diff
@@ -331,6 +363,21 @@ export default class EthereumService {
     throw new Error("Expected return or throw from attempt loop");
   }
 
+  async GasConfig() {
+    const block = await this.wallet.provider.getBlock("latest");
+    const previousBaseFee = block.baseFeePerGas;
+    assert(previousBaseFee !== null && previousBaseFee !== nil);
+
+    return {
+      maxFeePerGas: previousBaseFee
```

Comment:
Could be useful to have a comment that mentions why we set maxFeePerGas to the previousBaseFee + a fee increase + a priority fee per gas.

Reply:
Fixed in https://github.com/web3well/bls-wallet/pull/438